ui/src/components/WMErrorsTable/index.jsx

```diff
@@ -0,0 +1,276 @@
+      drawerError: null,
+    });
+  };
+
+  handleDrawerOpen = ({ currentTarget: { name } }) => {
+    const { errorsConnection } = this.props;
+    const drawerError = errorsConnection.edges.find(
+      ({ node }) => node.errorId === name
+    ).node;
+
+    this.setState({
+      drawerError,
+    });
+  };
+
+  renderTableRow = error => {
+    const { classes } = this.props;
+    const { errorId, title, description, reported } = error.node;
+    const iconSize = 16;
+
+    return (
+      <TableRow key={errorId}>
+        <TableCell>
+          <TableCellItem>
+            <ListItemText disableTypography primary={title} />
+          </TableCellItem>
+        </TableCell>
+        <TableCell>
+          <IconButton
+            className={classes.infoButton}
+            name={errorId}
+            onClick={this.handleDrawerOpen}>
+            <InformationVariantIcon size={iconSize} />
+          </IconButton>
+          <Typography className={classes.errorDescription}>
+            {description}
+          </Typography>
+        </TableCell>
+
+        <CopyToClipboard title={`${reported} (Copy)`} text={reported}>
+          <TableCell>
+            <TableCellItem button>
+              <ListItemText
+                disableTypography
+                primary={
+                  <Typography>
+                    <DateDistance from={reported} />
+                  </Typography>
+                }
+              />
+              <ContentCopyIcon size={iconSize} />
+            </TableCellItem>
+          </TableCell>
+        </CopyToClipboard>
+      </TableRow>
+    );
+  };
+
+  render() {
+    const { classes, errorsConnection, searchTerm, onPageChange } = this.props;
+    const { sortBy, sortDirection, drawerError } = this.state;
+    const sortedErrors = this.sortErrors(
+      errorsConnection,
+      sortBy,
+      sortDirection,
+      searchTerm
+    );
+
+    return (
+      <Fragment>
+        <ConnectionDataTable
+          connection={sortedErrors}
+          pageSize={VIEW_WORKER_POOL_ERRORS_PAGE_SIZE}
+          sortByHeader={sortBy}
+          sortDirection={sortDirection}
+          onHeaderClick={this.handleHeaderClick}
+          renderRow={this.renderTableRow}
+          headers={['Title', 'Description', 'Reported']}
+          padding="dense"
+          onPageChange={onPageChange}
+        />
+        <Drawer
+          anchor="right"
+          open={Boolean(drawerError)}
+          onClose={this.handleDrawerClose}
+          classes={{ paper: classes.drawerPaper }}>
+          {drawerError && (
+            <Fragment>
+              <IconButton
+                onClick={this.handleDrawerClose}
+                className={classes.drawerCloseIcon}>
+                <CloseIcon />
+              </IconButton>
+              <div className={classes.metadataContainer}>
+                <Typography variant="h5" className={classes.headline}>
+                  {drawerError.errorId}
+                </Typography>
```

Comment:
<img width="434" alt="Screen Shot 2019-09-27 at 4 13 26 PM" src="https://user-images.githubusercontent.com/3766511/65799298-cc67f680-e141-11e9-8aea-3d340e53ca21.png">

We can make this better by adding `text-overflow: ellipsis;`, `overflow: hidden;` and then setting a `min-width` to something a bit smaller than the width of the drawer.

<img width="367" alt="Screen Shot 2019-09-27 at 4 11 44 PM" src="https://user-images.githubusercontent.com/3766511/65799308-d2f66e00-e141-11e9-86a5-f87d3bcc6d39.png">

Also we can add `title={drawerError.errorId}` to show a tooltip when the user hovers on it just in case they are interested in seeing the end of the error id.